Bricks/strings/printf.h

```diff
@@ -33,6 +33,9 @@ SOFTWARE.
 namespace current {
 namespace strings {
 
+#ifndef CURRENT_WINDOWS
```

Comment:
Ok, `__attributes__` is a GNU extension and should be supported by a GNU compiler, so the right ifdef here is
```
#ifdef __GNUC__
```